array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,19 @@
+/* exported drop */
+
+/*
+  - Create new array variable
+  - Loop through input array
+  - If current index is greater than or equal to input count,
+    - then push array at current index into new array var
+  - Return new array variable outside loop
+*/
+
+function drop(array, count) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i >= count) {
+      newArray.push(array[i]);
+    }
```

Comment:
You can start your `i` variable higher than `0` so you don't need an `if` statement.